modules/pam_succeed_if/pam_succeed_if.8.xml

```diff
@@ -275,6 +275,7 @@
     </para>
     <programlisting>
 auth required pam_succeed_if.so quiet user ingroup wheel
+auth required pam_succeed_if.so quiet user ingroup group1:group2
```

Comment:
This could be `ingroup wheel:root.` And the sentence above could be adjusted like `except the fallback to group 0 being only approximated by checking also the root group membership`.